### PR TITLE
feat: add support for czech language

### PR DESCRIPTION
**Pull request by opravil-jan**
Hello,

would it be possible to add support for Czech language. Is the PR okay or do I need make any other changes please?

Thank you.

### CODE REVIEW

File: packages/languages/cs/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#### yarn:
+
+`yarn add @zxcvbn-ts/language-cs`
+
+## Setup
+
+```js
+import { zxcvbn, zxcvbnOptions } from '@zxcvbn-ts/core'
+import zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
+import zxcvbnEnPackage from '@zxcvbn-ts/language-cs'
```
Comment (MrWook) [inline packages/languages/cs/README.md:21]: zxcvbnCsPackage

File: packages/languages/cs/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+const password = 'somePassword'
+const options = {
+  translations: zxcvbnEnPackage.translations,
```
Comment (MrWook) [inline packages/languages/cs/README.md:25]: zxcvbnCsPackage

File: packages/languages/cs/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  graphs: zxcvbnCommonPackage.adjacencyGraphs,
+  dictionary: {
+    ...zxcvbnCommonPackage.dictionary,
+    ...zxcvbnEnPackage.dictionary,
```
Comment (MrWook) [inline packages/languages/cs/README.md:29]: zxcvbnCsPackage

**Review by MrWook — COMMENTED** [review]
Hey, there are just some smaller things in the readme 👍

**Comment by opravil-jan** [comment]
Thanks for review. It should be everything okay now.

**Comment by opravil-jan** [comment]
That is the reason I put to README.md  links on sources. From 2017 our government does not give the lists to public. We are negotiating with The Ministry of the Interior to give us a lists with actual data for year 2021, but we can not guarantee that will happened.  What should I do or would by the next step to take?

**Comment by MrWook** [comment]
I don't want that zxcvbn-ts is the single source of true for language sources so i really would like you to use the data-scripts folder to generate the language-package.
I saw that your sources are just html files. There is already a html parser for the `pt-br` language. You just need to add the extractor function like in the other language to parse the names from the html page.
To have `diacritics` and non `diacritics` you could add a parser to add both at the same time.

If you don't have the time or don't want to do it i can move those sources into the data-script folder.

Don't worry about the creation date of those name lists. It's better than nothing

**Pull request merged**